### ISSUE TITLE
The app does not boot on Android x86

### ISSUE THREAD

**Issue by falves1** [termux/termux-boot#39]
I need an ssh server to start with the box and bought termux-boot, but it does not do what it has promised to do: boot with the box.
I don't want a refund, I want somebody to fix it. 
android-x86_64-8.1-r2.iso, the version x86_64

**Comment by Grimler91** [comment]
What exactly doesn't work? 
What does your boot script contain? 
Have you followed the instructions in the app/the wiki? 
https://wiki.termux.com/wiki/Termux:Boot

**Comment by falves1** [comment]
Thanks for writing
The script is just to start sshd
Exactly the same that you mention in your product, two lines.
I reboot the box and it does not get executed.
This Android x86 8.1, the latest version.
I can give you access to the box via any remote desktop software.
 
Yours
Federico
 
 
--------- Original Message --------- Subject: Re: [termux/termux-boot] The app does not boot on Android x86 (#39)

**Comment by Grimler91** [comment]
Which two lines? 

Do you have a shebang? 
If not, add `#!/data/data/com.termux/files/usr/bin/sh`.
If yes, make sure it points to an existing shell in termux's prefix. 

Have you opened the termux-boot app as the instructions say?

**Comment by falves1** [comment]
yes, I followed your instructions verbatim.
let me try again
 
 
--------- Original Message --------- Subject: Re: [termux/termux-boot] The app does not boot on Android x86 (#39)

**Comment by falves1** [comment]
It works now.
You should urgently change the description that shows in the app, where you write
two lines..
and instead show three lines, the two plus the shebang
 
That was my mistake
 
Yours
 
Federico
 
--------- Original Message --------- Subject: Re: [termux/termux-boot] The app does not boot on Android x86 (#39)

**Comment by Grimler91** [comment]
Thanks for checking!
I've pushed an update to change the instructions: https://github.com/termux/termux-boot/commit/01b248629a7146abd91e787923f1ff3fefde1335 but a new app version probably won't be released before next week.

**Comment by falves1** [comment]
I need your advice on a VNC servet for my Android box, Android x86, rooted
 Strange enough, I cannot find one that works and has suppprt. Therenis a paid one but the comments say that the develploper does not respond to messages
 
Yours
Federico
--------- Original Message --------- Subject: Re: [termux/termux-boot] The app does not boot on Android x86 (#39)
